icon/chainscore_iiss.go

```diff
@@ -756,6 +769,7 @@ func (s *chainScore) Ex_setNetworkScore(role string, address module.Address) err
 			return icmodule.IllegalArgumentError.Errorf("Only scores owned by governance can be designated")
 		}
 	}
+	iiss.EmitNetworkScoreSetEvent(cc, role, address)
```

Comment:
In case where address is nil, it also works well, right?

Reply:
It is a Network SCORE deallocation case. `icx_getTransactionResult` responds with json `null`.